TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/RoutesToProfessionalStatus/EditRoute/CheckYourAnswers.cshtml.cs

```diff
@@ -154,23 +154,40 @@ await fileService.GetFileUrlAsync(ChangeReasonDetail.EvidenceFileId!.Value, File
 
     private bool IsComplete(RouteToProfessionalStatus route, ProfessionalStatusStatus status)
     {
-        return (QuestionDriverHelper.FieldRequired(route.TrainingStartDateRequired, status.GetStartDateRequirement()) != FieldRequirement.Mandatory
-            || JourneyInstance!.State.TrainingStartDate is not null) &&
-            (QuestionDriverHelper.FieldRequired(route.TrainingEndDateRequired, status.GetEndDateRequirement()) != FieldRequirement.Mandatory
-            || JourneyInstance!.State.TrainingEndDate is not null) &&
-            (QuestionDriverHelper.FieldRequired(route.AwardDateRequired, status.GetAwardDateRequirement()) != FieldRequirement.Mandatory
-            || JourneyInstance!.State.AwardedDate is not null) &&
-            (QuestionDriverHelper.FieldRequired(route.DegreeTypeRequired, status.GetDegreeTypeRequirement()) != FieldRequirement.Mandatory
-            || JourneyInstance!.State.DegreeTypeId is not null) &&
-            (QuestionDriverHelper.FieldRequired(route.TrainingAgeSpecialismTypeRequired, status.GetAgeSpecialismRequirement()) != FieldRequirement.Mandatory
-            || (JourneyInstance!.State.TrainingAgeSpecialismRangeFrom is not null && JourneyInstance!.State.TrainingAgeSpecialismRangeTo is not null) || JourneyInstance!.State.TrainingAgeSpecialismType is not null) &&
-            (QuestionDriverHelper.FieldRequired(route.TrainingCountryRequired, status.GetCountryRequirement()) != FieldRequirement.Mandatory
-            || JourneyInstance!.State.TrainingCountryId is not null) &&
-            (QuestionDriverHelper.FieldRequired(route.TrainingSubjectsRequired, status.GetSubjectsRequirement()) != FieldRequirement.Mandatory
-            || !JourneyInstance!.State.TrainingSubjectIds.IsNullOrEmpty()) &&
-            (QuestionDriverHelper.FieldRequired(route.TrainingProviderRequired, status.GetTrainingProviderRequirement()) != FieldRequirement.Mandatory
-            || JourneyInstance!.State.TrainingProviderId is not null) &&
-            (QuestionDriverHelper.FieldRequired(route.InductionExemptionRequired, status.GetInductionExemptionRequirement()) != FieldRequirement.Mandatory
-            || JourneyInstance!.State.IsExemptFromInduction is not null);
+        NotCompleteError errors = (QuestionDriverHelper.FieldRequired(route.TrainingStartDateRequired, status.GetStartDateRequirement()) != FieldRequirement.Mandatory
```

Comment:
Could this be written more cleanly maybe as a single statement ("or"ing with the None enum) something like this:
https://github.com/DFE-Digital/teaching-record-system/blob/main/TeachingRecordSystem/src/TeachingRecordSystem.Core/Services/WorkforceData/TpsCsvExtractProcessor.cs#L426